Update health assessment interval (#676)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/health.mdx

- [Drift detection](#drift-detection) determines whether your real-world infrastructure matches your Terraform configuration.
- [Continuous validation](#continuous-validation) determines whether custom conditions in the workspace’s configuration continue to pass after Terraform provisions the infrastructure.

> When enabled, HCP Terraform periodically runs an automated health assessment for your workspace. Refer to [Health Assessment Scheduling](#health-assessment-scheduling) for details.

Reviewer: ```suggestion
When you enable health assessments, HCP Terraform periodically runs health assessments for your workspace. Refer to [Health Assessment Scheduling](#health-assessment-scheduling) for details.
```

Remove passive voice, mention the feature by name. Although now it says "health assessment" twice, I think it scans ok?

## Permissions

Working with health assessments requires the following permissions:

- To view health status for a workspace, you need read access to that workspace.
- To change organization health settings, you must be an [organization owner](/terraform/cloud-docs/users-teams-organizations/permissions#organization-owners).
- To change a workspace’s health settings, you must be an [administrator for that workspace](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-admins).
<!-- BEGIN: TFC:only name:health-assessments -->

> - To trigger [on-demand assessments](/terraform/cloud-docs/workspaces/health#on-demand-assessments) for a workspace, you must be an [administrator for that workspace](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-admins).

Reviewer: ```suggestion
- To trigger [on-demand assessments](/terraform/cloud-docs/workspaces/health#on-demand-assessments) health assessments for a workspace, you must be an [administrator for that workspace](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-admins).
```

1. Select **Enable** under **Health Assessments**.
1. Click **Save settings**.

## Health assessment scheduling

> The timing of the first health assessment in the workspace depends on whether you enable health assessments during active Terraform runs:

Reviewer: ```suggestion
When you enable health assessments for a workspace, Terraform Cloud will run the first health assessment based on whether there are active Terraform runs for the workspace:
```

Reviewer: Remove passive voice.

> - **No active runs:** The first health assessment starts a few minutes after you enable the feature.
> - **Active speculative plan:** The first health assessment starts soon after that plan's completion.

Reviewer: ```suggestion
- **No active runs:** A few minutes after you enable the feature.
- **Active speculative plan:** A few minutes after that plan is complete.
```

> - **Other active runs:** The first health assessment starts during the next assessment period.

Reviewer: ```suggestion
- **Other active runs:** During the next assessment period.
```

A health assessment never interrupts or interferes with runs. If you start a new run during a health assessment, HCP Terraform cancels the current assessment and runs the next assessment during the next assessment period. This behavior may prevent HCP Terraform from performing health assessments in workspaces with frequent runs.

HCP Terraform pauses health assessments if the latest run ended in an errored state. This behavior occurs for all run types, including plan-only runs and speculative plans. Once the workspace completes a successful run, HCP Terraform restarts health assessments during the next assessment period.

Terraform Enterprise administrators can modify their installation's [assessment frequency  and number of maximum concurrent assessments](/terraform/enterprise/admin/application/general#health-assessments) from the admin settings console.

<!-- BEGIN: TFC:only name:health-assessments -->

### On-demand assessments

> -> **Note:** On-demand assessments are only available in the HCP Terraform web UI.

Reviewer 2: I like spelling out acronyms when we can:
```suggestion
-> **Note:** On-demand assessments are only available in the HCP Terraform user interface.
```

If you are an administrator for a workspace and it satisfies all [assessment requirements](/terraform/cloud-docs/workspaces/health#workspace-requirements), you can trigger a new assessment by clicking **Start health assessment** on the workspace's **Health** page.

> When you click the **Start health assessment** button, the workspace displays a message in the bottom lefthand corner of the page to indicate if it successfully triggered a new assessment. The time it takes to complete an assessment can vary based on network latency and the number of resources managed by the workspace. 

Reviewer 2: ```suggestion
After clicking **Start health assessment**, the workspace displays a message in the bottom lefthand corner of the page to indicate if it successfully triggered a new assessment. The time it takes to complete an assessment can vary based on network latency and the number of resources managed by the workspace. 
```

> While an assessment is in progress, you cannot trigger another assessment. An on-demand assessment resets the scheduling for automated assessments, so the next automated assessment will during the next assessment period. 

Reviewer: ```suggestion
While an assessment is in progress, you cannot trigger another assessment. An on-demand assessment resets the scheduling for automated assessments, so Terraform Cloud will run the next assessment during the next assessment period. 
```

> While an assessment is in progress, you cannot trigger another assessment. An on-demand assessment resets the scheduling for automated assessments, so HCP Terraform will run the next assessment during the next assessment period. 

Reviewer 2: Little tweaks for present tense
```suggestion
You cannot trigger another assessment while one is in progress. An on-demand assessment resets the scheduling for automated assessments, so HCP Terraform waits to run the next assessment until the next scheduled period.
```
